src_assets/common/assets/web/Navbar.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+    <nav class="navbar navbar-expand-lg navbar-light" style="background-color: #ffc400">
+        <div class="container-fluid">
+            <a class="navbar-brand" href="/" title="Sunshine">
+                <img src="/images/logo-sunshine-45.png" height="45" alt="Sunshine">
+            </a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
+                aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                    <li class="nav-item">
+                        <a class="nav-link" href="/"><i class="fas fa-fw fa-home"></i> Home</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/pin"><i class="fas fa-fw fa-unlock"></i> PIN</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/apps"><i class="fas fa-fw fa-stream"></i> Applications</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/config"><i class="fas fa-fw fa-cog"></i> Configuration</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/password"><i class="fas fa-fw fa-user-shield"></i> Change Password</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/troubleshooting"><i class="fas fa-fw fa-info"></i> Troubleshooting</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+</template>
+
+<script>
+export default {
+    created() {
+        console.log("Header mounted!")
+    },
+    mounted() {
+        let el = document.querySelector("a[href='" + document.location.pathname + "']");
+        if (el) el.classList.add("active")
+        let widgetbot = document.createElement('script')
+        widgetbot.setAttribute('src', 'https://cdn.jsdelivr.net/npm/@widgetbot/crate@3')
+        widgetbot.async = true;
+        widgetbot.onload = () => {
+            console.log("Widgetbot script created, starting up")
+            new Crate({
+                server: '804382334370578482',
+                channel: '804383092822900797',
+                defer: false,
+            })
+        }
+        document.head.appendChild(widgetbot)
```

Comment:
//this script requires jquery to be loaded on the source page, like so...

I'm not a fan of adding jQuery for a completelty unrelated script, and there's also some unrelated code regarding the quote system which downloads other JSON files. Can we split this into two separate files?

Reply:
The quote system is part of the reason to use this script.

Using it will make it so we can update the discord widget in one repo in LizardByte instead of 6 different repos.

Reply:
Gotcha, can I migrate this to VanillaJS then?

Reply:
Also, any reason for sleep.js instead of just including the function directly?

Reply:
> Also, any reason for sleep.js instead of just including the function directly?

Same reason for the discord.js, I didn't want to write a sleep function in many repos.

Reply:
Just for anyone following the convo, link for the PR on the website side: https://github.com/LizardByte/LizardByte.github.io/pull/155

Reply:
This is now merged.